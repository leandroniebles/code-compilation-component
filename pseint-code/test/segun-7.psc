proceso prueba
	si verdadero entonces
		escribir 'a';
	de otro modo: 
		escribir 'b';
	finsi
	segun 1 hacer
		1: escribir 'a';
	sino: 
		escribir 'b';
	finsegun
finproceso
