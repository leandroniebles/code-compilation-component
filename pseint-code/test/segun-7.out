=== Line 4: SynError 321
=== Line 9: SynError 2
